### PR TITLE
Move on start server

### PR DESCRIPTION
**Pull request by uweenukr**

### CODE REVIEW

File: Assets/Mirror/Runtime/NetworkServer.cs [hunk 1]
```diff
@@ -35,6 +35,13 @@ public class NetworkServer : MonoBehaviour
         public NetworkConnectionEvent Connected = new NetworkConnectionEvent();
         public NetworkConnectionEvent Authenticated = new NetworkConnectionEvent();
         public NetworkConnectionEvent Disconnected = new NetworkConnectionEvent();
+
+        /// <summary>
+        /// This is invoked when a server is started - including when a host is started.
+        /// <para>StartServer has multiple signatures, but they all cause this hook to be called.</para>
+        /// </summary>
+        public UnityEvent OnStartServer = new UnityEvent();
```
Comment (paulpach) [inline Assets/Mirror/Runtime/NetworkServer.cs]: Rename to "Started". for consistency with the other events.

We are already in the NetworkServer class,  so the word "Server" is redundant.

Also move above the other events,  this is the first one that gets fired

File: Assets/Mirror/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
+## [29.0.1](https://github.com/MirrorNG/MirrorNG/compare/29.0.0-master...29.0.1-master) (2020-03-27)
```
Comment (paulpach) [inline Assets/Mirror/CHANGELOG.md]: YOu should not modify the CHANGELOG yourself,

This file is autogenerated by the CI/CD pipeline.

File: Assets/Mirror/Runtime/NetworkManager.cs [hunk 3]
```diff
@@ -906,18 +896,5 @@ public virtual void OnClientSceneChanged(NetworkConnectionToServer conn)
 
         #endregion
 
-        #region Start & Stop callbacks
```
Comment (paulpach) [inline Assets/Mirror/Runtime/NetworkManager.cs]: beautiful!

File: Assets/Mirror/CHANGELOG.md (hunk 2 above)
Comment (uweenukr) [reply Assets/Mirror/CHANGELOG.md]: That wasn't something I changed. When I was trying to catch this branch up I'm pretty sure I did it wrong.

Comment (uweenukr) [reply Assets/Mirror/CHANGELOG.md]: fixed

File: Assets/Mirror/Runtime/NetworkServer.cs (hunk 1 above)
Comment (uweenukr) [reply Assets/Mirror/Runtime/NetworkServer.cs]: fixed

File: Assets/Mirror/Runtime/NetworkManager.cs (hunk 3 above)
Comment (uweenukr) [reply Assets/Mirror/Runtime/NetworkManager.cs]: it was very satisfying. wish you could have been there.

**Pull request merged**